src/stores/AppStore.ts

```diff
@@ -470,6 +464,12 @@ export class AppStore {
     };
 
     public static readonly DEFAULT_STATS_TYPES = [CARTA.StatsType.NumPixels, CARTA.StatsType.Sum, CARTA.StatsType.Mean, CARTA.StatsType.RMS, CARTA.StatsType.Sigma, CARTA.StatsType.SumSq, CARTA.StatsType.Min, CARTA.StatsType.Max];
+    private static readonly CursorThrottleTime = 200;
```

Comment:
constants better be capitalized. Btw, what is the reason to move those constants to private members?

Reply:
according to the AirBnB style guide, we should only do exported consts in `ALL_CAPS`. I know we're not consistent about this, but perhaps it's better to work toward that. 

We currently use `PascalCase` instead of `camelCase` for a lot of other `static readonly` variables, so I've tried to be consistent with that. Perhaps this needs a bit more thought though. One solution which is somewhat consistent is to use `ALL_CAPS` for `public static readonly`, since those behave like `export const` values, and `camelCase` (or `PascalCase` if we don't want to adjust existing code too much) for `private static readonly`

It would be good to get opinions on that from @panchyo0 as well. 

As for why I've moved the const to private members: they're not used outside of the class, so I thought it would be better to keep them in the class as private static values.

Reply:
"consistent is to use ALL_CAPS for public static readonly, since those behave like export const values, and PascalCase for private static readonly." This one might better for our next release.

Reply:
> This one might better for our next release.

Ok, I'll make an issue for this and tag it for 1.3. It should be quite easy to fix with an IDE